packages/block-library/src/form-input/style.scss

```diff
@@ -0,0 +1,57 @@
+.wp-block-form-input__label {
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	gap: 0.25em;
+	margin-bottom: 0.5em;
+
+	&.is-label-inline {
+		flex-direction: row;
+		gap: 0.5em;
+		align-items: center;
+	}
+
+	/*
+	Small tweak to left-align the checkbox.
+	Even though `:has` is not currently supported in Firefox, this is a small tweak
+	and does not affect the functionality of the block or the user's experience.
+	There will be a minor inconsistency between browsers. However, it's more important to provide
+	a better experience for 80+% of users, until Firefor catches up and supports `:has`.
```

Comment:
```suggestion
	a better experience for 80+% of users, until Firefox catches up and supports `:has`.
```
Small typo.

Reply:
Should we link to https://caniuse.com/css-has here so that we can easily check if Firefox has added support in the future?